Description: remove code that was overriding debian default compilation flags
 dcmtk upstream was forcing us to use -O3 and -NDEBUG which is EBW !
 See thread at: http://bugs.debian.org/711515
Author: Mathieu Malaterre <malat@debian.org>
Forwarded: no

Index: dcmtk-3.6.1~20131114/CMake/dcmtkPrepare.cmake
===================================================================
--- dcmtk-3.6.1~20131114.orig/CMake/dcmtkPrepare.cmake	2013-11-27 15:52:05.000000000 +0100
+++ dcmtk-3.6.1~20131114/CMake/dcmtkPrepare.cmake	2013-11-27 15:52:41.000000000 +0100
@@ -19,12 +19,12 @@
 # Debug and Release). It is empty for other generators and for those
 # generators the build type is controlled at CMake time by CMAKE_BUILD_TYPE.
 # http://www.cmake.org/pipermail/cmake/2006-January/008065.html
-IF(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-  MESSAGE(STATUS "Setting build type to 'Release' as none was specified.")
-  SET(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
-  # Set the possible values of build type for cmake-gui
-  SET_PROPERTY(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
-ENDIF(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+#IF(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+#  MESSAGE(STATUS "Setting build type to 'Release' as none was specified.")
+#  SET(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+#  # Set the possible values of build type for cmake-gui
+#  SET_PROPERTY(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+#ENDIF(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
 
 
 # Basic version information
